.github/workflows/run-merge-hive-tests.yml

```diff
@@ -1,31 +1,9 @@
-name: '[RUN] Hive Tests'
+name: '[RUN] Hive tests'
    
 on: 
   push:
     tags:
     - '*'
-    inputs:
-      testSuite:
-        description: 'Test suite'
-        required: true
-        default: 'ethereum/engine' 
-        type: choice
-        options:
-        - ethereum/engine        
-      limit:
-        description: 'Limit for hive tests'
-        required: false
-        type: string
-      hive:
-        description: 'Repository for hive tests'
-        required: true
-        default: 'ethereum/hive'
-        type: string        
-      branch:
-        description: 'Branch for hive tests'
-        default: 'master'
-        required: false
-        type: string
```

Comment:
Will this work well on pushing tags, without those?

Reply:
Those were redundant and should not be there. The right ones are just below.